other/dns-policy-and-dns-config/dns-policy-and-dns-config.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: dns-policy
+  annotations:
+    policies.kyverno.io/title: Change DNS Config and Policy
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    kyverno.io/kyverno-version: 1.7.3
+    kyverno.io/kubernetes-version: "1.23"
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
```

Comment:
You may also want to state this policy requires a kubeadm-provisioned cluster to operate properly. Remember that not all clusters are built using kubeadm or even have the same resources available.

Reply:
Yes, you are right, I fixed the description.